indico/modules/events/editing/operations.py

```diff
@@ -241,7 +241,8 @@ def create_new_file_type(event, **data):
 
 
 def update_file_type(file_type, **data):
-    file_type.populate_from_dict(data, keys=('name', 'extensions', 'allow_multiple_files', 'required', 'publishable'))
+    attrs_to_update = ('name', 'extensions', 'allow_multiple_files', 'required', 'publishable', 'filename_template')
```

Comment:
shouldn't this be updated in `create_new_file_type` as well? (maybe move the list to the top level and use it in both places)

Reply:
Indeed, I only tested update process :P